nservicebus/transports/guidance.md

```diff
@@ -0,0 +1,71 @@
+---
+title: SQL Server Transport usage guidance
+summary: Usage guidance for SQL Server transport
+tags:
+- SQL Server
+---
+
+The SQL Server transport allows endpoints to exchange messages using the Microsoft SQL Server Database Engine. It implements queues on top of database tables and send / receive operations as `INSERT` and `DELETE` operations.
+
+Please note that the SQL Server transport does not use or depend on SQL Server Service Broker.
+
+## How it works
+
+SQL Server provides a central place to store queues and messages but the message queuing implementation resides entirely within the endpoint process. SQL Server transport is best thought of as a brokered transport like RabbitMQ rather than store-and-forward transport such as MSMQ.
```

Comment:
Maybe explain the difference between a broker style architecture to a distributed one, mentioning the single point of failure? Also the Store and Forward mechanism (maybe a link to a document describing store and forward in MSMQ?)